docs/NativeModulesAndroid.md

```diff
@@ -18,7 +18,7 @@ This guide will use the [Toast](http://developer.android.com/reference/android/w
 We start by creating a native module. A native module is a Java class that usually extends the `ReactContextBaseJavaModule` class and implements the functionality required by the JavaScript. Our goal here is to be able to write `ToastAndroid.show('Awesome', ToastAndroid.SHORT);` from JavaScript to display a short toast on the screen.
 
 ```java
-package com.facebook.react.modules.toast;
+package com.example.modules.toast;
```

Comment:
I think we should. because if someone is moving is existing native android app files to react, he would'nt be doing creating new files rather just moving them here.
But what we can do is, be explicit stating your "package name" is the app name 
package com.<appname> 
 Also same issue here too 
https://github.com/facebook/react-native/issues/3515